migrations/20171104220247_add_heilas_table.js

```diff
@@ -0,0 +1,27 @@
+
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('heilas', function(table) {
+      table.integer('userId').notNullable().unique().primary().index();
+      table.foreign('userId')
+        .references('id')
+        .inTable('users')
+        .onDelete('RESTRICT')
+        .onUpdate('CASCADE')
+
+      table.string('uuid').notNullable().unique().index();
+      table.foreign('uuid')
+        .references('uuid')
+        .inTable('users')
+        .onDelete('RESTRICT')
+        .onUpdate('CASCADE');
+
+      table.string('bio_text', 250);
+      table.string('bio_looking_for', 250);
```

Comment:
Inside the same table we have two different styles for writing column names: snake_case and camelCase. Consistency is the name of the game, we should probably stick to the method used in prior tables

Reply:
True. Thx.

Reply:
@cc-akarvinen